addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1725,12 +1730,24 @@ const RangeUserValueWidget = UnitUserValueWidget.extend({
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     */
+    getMethodsParams(methodName) {
+        const params = this._super(...arguments);
+        return this.isCustomRange ?
+        {
+            ...params,
```

Comment:
De-structuring, with rest in object is still considered an experimental feature by caniuse https://caniuse.com/mdn-javascript_operators_destructuring_rest_in_objects. Besides, this would be way clearer and probably even more efficient to write this this way:

```javascript
const params = this._super(...arguments);
if (this.isCustomRange) {
     params.possibleValues = ___;
}
return params;